examples/chat.sh

```diff
@@ -11,6 +11,6 @@ cd ..
 #
 #   "--keep 48" is based on the contents of prompts/chat-with-bob.txt
 #
-./main -m ./models/llama-7b/ggml-model-q4_0.gguf -c 512 -b 1024 -n 256 --keep 48 \
+./main -m ./models/7B/ggml-model-q4_0.gguf -c 512 -b 1024 -n 256 --keep 48 \
```

Comment:
Better to update the readme to `llama-7b`
The name `7B` is ambiguous nowadays

Reply:
Will do.  I'll close this one out and open a new PR for readme change.